Configuration/Eras/python/Modifier_pps_2016_cff.py

```diff
@@ -0,0 +1,4 @@
+import FWCore.ParameterSet.Config as cms
+
+pps_2016 =  cms.Modifier()
```

Comment:
There is another ctpps_2016 modifier in Configuration/Eras/python/Modifier_ctpps_2016_cff.py: please avoid duplications, if not necessaries

Reply:
Why aren't these modifiers included in the Eras, e.g. Run2_2016 era as it is now the ctpps_2016 modifier in Configuration/Eras/python/Era_Run2_2016_cff.py?

Reply:
A change done in the past in this config broke the reconstruction somewhere else. A deeper investigation is needed here. If it can be kept like this for the moment, we would appreciate it.

Reply:
Could you please elaborate? 
Do you plan to use the ```pps_2016``` era independently on ```Run2_2016```? What will be wrong with unifying  ```ctpps_2016```  and  ```pps_2016``` ?